### DISCUSSION TITLE
tsconfig paths alias for esm not work

### DISCUSSION THREAD

**Discussion by pincman** [TypeStrong/ts-node#1807] [General]
```json
  // ...
   "paths": {
            "@/*": ["./src/*"]
        }
    },
    "ts-node": {
        "require": ["tsconfig-paths/register"]
    },
```
it not work for esm project,but work fine with commandjs

**Comment by cspotcode** [comment]
Tsconfig-paths doesn't support ESM

**Reply by pincman** [reply]
Is there any solution for this problem?

**Comment by gudh** [comment]
any updates?

Same problem here

**Comment by wvhulle** [comment]
I also have this problem. See my question https://stackoverflow.com/posts/74731164

**Comment by anaibol** [comment]
Use https://github.com/digital-loukoum/esrun
